docs/1-0-preview.md

```diff
@@ -31,13 +31,13 @@ Workflow executions will fail when there are syntax errors or runtime errors in
 
 This makes [user management](/user-management/) mandatory, and removes support for other methods such as BasicAuth and JWT. It will include a change to n8n Cloud pricing plans to make user management is available to all tiers. Some tiers may still be limited to one user.
 
-### Remove support for MySQL and MariaDB as n8n backend databases
+### Deprecation of MySQL and MariaDB as n8n backend databases
 
-The default database will remain SQLite. Postgres is the recommended alternative.
+Usage of MySQL or MariaDB is disencouraged and can be removed in the future. Support will continue after vv1 has been released but we recommend you migrating to PostgreSQL.
 
 ### Main mode as default
 
-n8n will now run in main mode by default. Refer to [Execute all workflows in the same process](/hosting/environment-variables/configuration-methods/#execute-all-workflows-in-the-same-process) for more information on own and main mode. You should use [Queue mode](/hosting/scaling/queue-mode/) if you need scalability.
+n8n will now run in `main` mode by default and `own` mode will be completely removed. Refer to [Execute all workflows in the same process](/hosting/environment-variables/configuration-methods/#execute-all-workflows-in-the-same-process) for more information on own and main mode. You should use [Queue mode](/hosting/scaling/queue-mode/) if you need scalability.
```

Comment:
```suggestion
Starting from v1.0, n8n will operate in `main` mode as the default option, and `own` mode will eventually be phased out. For detailed insights on `own` and `main` mode, please refer to the section on [Executing all workflows in the same process](/hosting/environment-variables/configuration-methods/#execute-all-workflows-in-the-same-process). If scalability is a requirement, it is recommended to utilize the [Queue mode](/hosting/scaling/queue-mode/).
```

Reply:
About this topic, we're completely removing `own` mode, so maybe we should leave it clear that `main` is the default and only mode and if you want scalability, consider `queue`. WDYT?

Your suggestion says that `own` will eventually be phased out but it's actually planned for full removal in 1.0

Reply:
Wait, as far as I'm aware we're only deprecating `own` mode and making `main` the default. We're not removing `own` yet.